rollup.config.js

```diff
@@ -1,27 +1,31 @@
-import nodeResolve from 'rollup-plugin-node-resolve'
-import babel from 'rollup-plugin-babel'
-import { uglify } from 'rollup-plugin-uglify'
+import nodeResolve from '@rollup/plugin-node-resolve'
+import babel from '@rollup/plugin-babel'
+import { terser } from 'rollup-plugin-terser'
```

Comment:
The single quotes have been using. What about including `.js` to the target in ESLint?